web/tests/stream_data.test.js

```diff
@@ -496,15 +496,17 @@ test("delete_sub", () => {
     };
 
     stream_data.add_sub(canada);
+    const num_subscribed_subs = stream_data.num_subscribed_subs();
 
     assert.ok(stream_data.is_subscribed_by_name("Canada"));
     assert.equal(stream_data.get_sub("Canada").stream_id, canada.stream_id);
     assert.equal(sub_store.get(canada.stream_id).name, "Canada");
 
     stream_data.delete_sub(canada.stream_id);
     assert.ok(!stream_data.is_subscribed_by_name("Canada"));
-    assert.ok(!stream_data.get_sub("Canada"));
-    assert.ok(!sub_store.get(canada.stream_id));
+    assert.ok(stream_data.get_sub("Canada"));
+    assert.ok(sub_store.get(canada.stream_id));
+    assert.equal(stream_data.num_subscribed_subs(), num_subscribed_subs - 1);
```

Comment:
So just add a couple calls here to check whether the Canada stream is archived before and after deactivating it, and this test will have 100% coverage of `stream_data.ts` again.

Reply:
Done.